sys/phydat/phydat.c

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2018 Eistec AB
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     driver_sensif
```

Comment:
weird, this group is not defined (git grep only return an `@ingroup driver_sensif` in phydat_str.c. Looks like there a typo somewhere or something missing.

Reply:
addressed in #8592